controllers/storagecluster/cephcluster.go

```diff
@@ -203,6 +205,17 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		cephCluster.Spec.HealthCheck.DaemonHealth.Monitor.Timeout = "15m"
 	}
 
+	IPFamily, isDualStack, err := getIPFamilyConfig(r.Client)
```

Comment:
We are always getting false for isDualStack var, So is it necessary to be returned by the functions, can't we just set it here?

Reply:
We are always setting it to `false` because we currently don't support dualStack downstream. Rook supports it upstream though. I've added the details in the `getIPFamilyConfig` function. 

In future if we plan to support DualStack downstream as well, then we should return `true` for isDualStack if the OCP cluster is dual stack.